File: ChatDemo-UI3.0/ChatDemo-UI3.0/Code/Controllers/Settings/EMAboutViewController.m
/************************************************************
 *  * Hyphenate
 * __________________
 * Copyright (C) 2016 Hyphenate Inc. All rights reserved.
 *
 * NOTICE: All information contained herein is, and remains
 * the property of Hyphenate Inc.
 */

#import "EMAboutViewController.h"

@interface EMAboutViewController ()

@end

@implementation EMAboutViewController

- (void)viewDidLoad
{
    [super viewDidLoad];
    [self configBackButton];
}


#pragma mark - Table view data source


- (NSInteger)tableView:(UITableView *)tableView numberOfRowsInSection:(NSInteger)section
{
    return 2;
}


- (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(NSIndexPath *)indexPath {
    static NSString *cellIndetifier = @"aboutCell";
    UITableViewCell *cell = [tableView dequeueReusableCellWithIdentifier:cellIndetifier];
    if (!cell) {

        cell = [[UITableViewCell alloc] initWithStyle:UITableViewCellStyleValue1 reuseIdentifier:cellIndetifier];
    }
    if (indexPath.row == 0) {
        
        cell.textLabel.text = NSLocalizedString(@"setting.about.appversion", @"App Version");
        cell.detailTextLabel.text = [[[NSBundle mainBundle] infoDictionary] objectForKey:@"CFBundleShortVersionString"];
    } else if (indexPath.row == 1) {
        
        cell.textLabel.text = NSLocalizedString(@"setting.about.sdkversion", @"SDK Version");
        cell.detailTextLabel.text = [[EMClient sharedClient] version];
    }
    
    return cell;
}


@end

File: swift-demo/Hyphenate-Demo-Swift/Code/Controllers/Settings/EMAboutViewController.swift
//
//  EMAboutViewController.swift
//  Hyphenate-Demo-Swift
//
//  Created by 杜洁鹏 on 2017/7/7.
//  Copyright © 2017年 杜洁鹏. All rights reserved.
//

import UIKit
import Hyphenate

class EMAboutViewController: UITableViewController {
    
    @IBOutlet weak var appVersionLabel: UILabel!
    @IBOutlet weak var sdkVersionLabel: UILabel!
    
    override func viewDidLoad() {
        super.viewDidLoad()
        tableView.tableFooterView = UIView()
        appVersionLabel.text = Bundle.main.infoDictionary?["CFBundleShortVersionString"] as? String
        sdkVersionLabel.text = EMClient.shared().version
    }
}
